### DISCUSSION TITLE
Original Colors preservation

### DISCUSSION THREAD

**Discussion by AlexDreico** [lllyasviel/ControlNet#179] [Ideas]
![Untitled-1](https://user-images.githubusercontent.com/8235292/221501678-6750b1a5-e04c-4314-bd7f-41622d9782a1.png)
 May you add on option or checkbox to save colors from original picture in img2img. Now the colors are changing because of great Denoising strength

**Comment by geroldmeisinger** [comment]
ControlNet color https://huggingface.co/thibaud/controlnet-sd21#color
